projects/igniteui-angular/src/lib/tabbar/tabbar.component.ts

```diff
@@ -177,12 +171,18 @@ export class IgxTabPanelComponent implements AfterContentInit {
         this._tabTemplate = template;
     }
 
+    get nativeElement() {
+        return this._nativeElement;
+    }
+
     private _tabTemplate: TemplateRef<any>;
+    private _nativeElement: ElementRef;
 
     @ContentChild(IgxTabTemplateDirective, { read: IgxTabTemplateDirective })
     protected tabTemplate: IgxTabTemplateDirective;
 
-    constructor(private _tabBar: IgxBottomNavComponent) {
+    constructor(private _tabBar: IgxBottomNavComponent, private _element: ElementRef) {
+        this._nativeElement = _element;
```

Comment:
You don't need this private field _nativeElement and the public get nativeElement property. Just use _element which is defined in the constructor.